docs/docs/concepts/tests/tests-creating.md

```diff
@@ -434,6 +434,22 @@ spec:
 
 When you run such a test you will face a memory limit for the scraper pod, when the default scraper job template doesn't have any resource constraints.
 
+### **Mounting config maps and secrets to executor pod**
+If you need to mount your config maps and secrets to your executor environment, then you can provide them as additional  
+parameters when you create or run the test using `--mount-configmap` and `--mount-secret` options:
+
+```bash
+kubectl testkube create test --file test/postman/LocalHealth.postman_collection.json --name mount-test --type postman/collection --mount-configmap your_configmap=/pod_mount_path --mount-secret your_secret=/pod_mount_path
+```
```

Comment:
```suggestion
```sh
testkube create test --file test/postman/LocalHealth.postman_collection.json --name mount-test --type postman/collection --mount-configmap your_configmap=/pod_mount_path --mount-secret your_secret=/pod_mount_path
```
```

Reply:
prefer sh over bash

prefer not using `kubectl testkube`, but using `testkube`